express-api/src/routes/notificationsRouter.ts

```diff
@@ -9,12 +9,20 @@ export const DISPOSAL_API_ROUTE = '/projects/disposal';
 export const NOTIFICATION_QUEUE_ROUTE = '/queue';
 export const NOTIFICATION_TEMPLATE_ROUTE = '/templates';
 
-const { getNotificationsByProjectId } = controllers;
+const { getNotificationsByProjectId, resendNotificationById, cancelNotificationById } = controllers;
 
 //I believe that the IDs used in these routes are actually the project ID, even though the structure here sort of implies
 //that it might be an individual "notification id".
 router
-  .route(`${NOTIFICATION_QUEUE_ROUTE}`)
+  .route(NOTIFICATION_QUEUE_ROUTE)
   .get(activeUserCheck, catchErrors(getNotificationsByProjectId));
 
+router
+  .route(`${NOTIFICATION_QUEUE_ROUTE}/:id`)
+  .put(activeUserCheck, catchErrors(resendNotificationById));
+
+router
+  .route(`${NOTIFICATION_QUEUE_ROUTE}/:id`)
+  .delete(activeUserCheck, catchErrors(cancelNotificationById));
```

Comment:
This feels sort of misleading, as it doesn't delete the record, but I think this is probably the right way to go.

Reply:
I feel similarly about using PUT for resend. The old implementation instead does a PUT to either /id/resend or /id/cancel but I think it's generally considered poor practice to make your endpoints verbs rather than just using the HTTP request type verbage.